dotcom-rendering/src/web/components/RichLink.tsx

```diff
@@ -39,8 +39,10 @@ export interface RichLinkImageData {
 	height: string;
 }
 
-const neutralBackground = css`
-	background-color: ${neutral[97]};
+const neutralBackground = (format: ArticleFormat) => css`
+	background-color: ${format.design === ArticleDesign.Analysis
```

Comment:
Nice work! I think we'll want to lift this up into palette to be consistent with DCR